### ISSUE TITLE
Documentation: Re-examining introduction tutorials

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#27136]
**---- Issue is being worked on. ----**

I am re-examining the what I would call introduction tutorials.

Focused on: Contributors, theme/plugin/etc developers and someone who wants to give feedback to Gutenberg.

The doorway would be this tutorial to setup a local WP development environment:
https://developer.wordpress.org/block-editor/tutorials/devenv/#quickstart

After setting up the above there are different directions depending on what one wants to accomplish.

1. I want to test PR's:
https://github.com/WordPress/gutenberg/issues/25202
(Issue is still being developed. I should end up with a few simple as possible testing methods.

2. I want to contribute to Gutenberg by submitting a PR. 
With code:
https://developer.wordpress.org/block-editor/contributors/develop/
With documentation:
https://developer.wordpress.org/block-editor/contributors/document/

3. I want to create a Gutenberg block.
https://developer.wordpress.org/block-editor/tutorials/block-tutorial/

4. I want to create a Gutenberg pattern.
https://developer.wordpress.org/block-editor/developers/block-api/block-patterns/

**NB. I will add on to the list, but need to focus on just a few tutorials at a time.**

Bottom line is a step by step approach is needed for most tutorials. I plan to re-examine the above in relation to simplifying steps where it is needed.

This issue touches very strongly on what Justin wrote about in these documentation posts:
https://make.wordpress.org/docs/2020/08/13/plan-proposal-for-a-new-better-structured-gutenberg-developer-documentation/
and
https://make.wordpress.org/docs/2020/11/12/follow-up-on-gutenberg-developer-documentation-restructuring-proposal/

**Comment by ryanwelcher** [comment]
@paaljoachim just wanted to follow up on this to see if it is still relevant.

**Comment by paaljoachim** [comment]
Thanks for the nudge Ryan!

I have not followed up on this issue for a long time so I will go ahead and close it.
